composer.json

```diff
@@ -39,7 +39,7 @@
     },
     "require": {
         "php": "^7.2 || ^8.0",
-        "babenkoivan/elastic-adapter": "^1.13"
+        "babenkoivan/elastic-adapter": "^1.16"
```

Comment:
We also need to require `babenkoivan/elastic-scout-driver: ^1.3.1` now, cause it will not work with other versions.